.github/workflows/wheel_win_x86_64.yml

```diff
@@ -50,8 +54,13 @@ jobs:
         with:
           python-version: '3.7'
 
+      - name: Configure MSVC for amd64 # Use cl.exe as a default compiler
+        uses: ilammy/msvc-dev-cmd@v1
+        with:
+          arch: amd64
+
```

Comment:
I have tried all possible combinations I can imagine today, but failed to make it works.... I just cannot make github actions to use `cmake` from MSVC instead of separately installed (which uses MinGW by default). With this `cmake`, even after I set the compiler explicitly (by setting `-DCMAKE_CXX_COMPILER)`, it cannot find correct linker, etc. Thus I hope to use `ilammy/msvc-dev-cmd` at the moment.

Reply:
Sure thing. We can plan to replace this a little later.